source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,273 @@
+For example, you may want to collect:
+
+* money each time a service is provided
+
+* payments where the amounts vary according to service usage
+
+* regular payments where you do not know in advance how much each payment will be
+
+You could also use the variable functionality to simulate a recurring or
+scheduled Direct Debit payment, by sending an API call on a set schedule.
+Users would be emailed each time a payment is collected against their mandate.
+
+You are responsible for managing variable payments using the GOV.UK Pay API.
+
+### Set up a variable Direct Debit payment
+
+1. Create an agreement using `POST /v1/agreements/`,
+specifying:
+
+* a `return_url` where GOV.UK Pay will direct the user when they have finished
+setting up the agreement
+
+* the `agreement_type` (`on demand`)
+
+* a `reference` (optional) - your reference for this agreement
+
+The response contains a `next_url`.
+
+2. Direct the user to the `next_url`.
+
+3. The user provides their payment details. For example, account number
+and sort code for Direct Debit. GOV.UK Pay makes an API call to our Direct
+Debit supplier to verify account details.
+
+4. GOV.UK Pay presents the user with a confirmation screen which presents
+their account details. The user grants their consent by selecting
+__Confirm__.  GOV.UK Pay sends an email to the user to confirm that their
+mandate has been set up. If the mandate subsequently fails, the GOV.UK Pay team
+will send an email to update the user that their mandate could not be set up.
+
+5. GOV.UK Pay securely passes the user's details on to the Direct Debit
+provider, who submit a request to BACS to set up a mandate.
+
+6. GOV.UK Pay returns the user to your service using the `return_url` from
+Step 1.
+
+7. Call `GET /v1/agreements/<AGREEMENT-ID>` to check the status of the
+agreement (in order to display an appropriate screen to the user). For a
+Direct Debit payment, this will initially be in a ‘pending’ state until the
+bank confirms the mandate. The confirmation process typically takes 3-4 days.
+
+8. When GOV.UK Pay receives confirmation that the mandate is
+confirmed, the GOV.UK Pay admin tool will update the transaction status.
+
+## Control the payment method
+
+When you create an API key, you choose which payment gateway that key is
+connected with. For example “testing”, “Worldpay”, or “Direct Debit”. You can
+use different API keys to direct paying users to different payment methods.
+
+### Agreement lifecycle
+
+1. `created` - Your service made an API call to create the agreement
+
+2. `started` - Payer has visited the `next_url` and landed on the GOV.UK Pay
+page to enter their payment details
+
+3. `pending` - Indicates that payment details have been submitted to the
+payment service provider (PSP), but the PSP is not yet able to confirm whether
+the payment has been successful
+
+4. The agreement reaches one of several finished states:
+
+* `success` - Payment successfully completed
```

Comment:
mandate completed - let's review that message from Andrew L together which contains amendments to this section